src/drive/styles/topbar.styl

```diff
@@ -13,5 +13,9 @@ $coz-bar-size = 3rem
     flex 0 0 auto
 
 +small-screen()
-    .fil-topbar
+  .fil-topbar
+    margin-right .5rem
+    margin-left .5rem
+
+  .fil-topbar.hidden-mobile
```

Comment:
Usually we hide the topbar on mobile, but inside this interapp modal we want to keep it displayed.